src/main/java/formflow/library/file/S3CloudFileRepository.java

```diff
@@ -38,21 +36,19 @@
 public class S3CloudFileRepository implements CloudFileRepository {
 
     private final String bucketName;
-    private final AmazonS3 s3Client;
-    private final TransferManager transferManager;
+    private final S3Client s3Client;
 
     public S3CloudFileRepository(@Value("${form-flow.aws.access_key}") String accessKey,
-                                 @Value("${form-flow.aws.secret_key}") String secretKey,
-                                 @Value("${form-flow.aws.s3_bucket_name}") String s3BucketName,
-                                 @Value("${form-flow.aws.region}") String region) {
-        AWSCredentials credentials = new BasicAWSCredentials(accessKey, secretKey);
+            @Value("${form-flow.aws.secret_key}") String secretKey,
+            @Value("${form-flow.aws.s3_bucket_name}") String s3BucketName,
+            @Value("${form-flow.aws.region}") String region) {
+
         bucketName = s3BucketName;
-        s3Client = AmazonS3ClientBuilder
-                .standard()
-                .withRegion(region)
-                .withCredentials(new AWSStaticCredentialsProvider(credentials))
+
+        AwsBasicCredentials awsCreds = AwsBasicCredentials.create(accessKey, secretKey);
```

Comment:
It's possible this might not work in QA, but I'm not sure. We may just have to make the change here and then see.